Adding all video links + keynotes

doctorrobinson opened the pull request:
Like it says. Also added playlists, some are still unlisted, but they'll still work once we go public!

doctorrobinson committed:
Adding all video links + keynotes

terichadbourne commented:
Haven't had a chance to proof this, but I just realized that when I added course video links to the ToC in the readme I should have also added them within the readme of each course's own folder (for example [https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/CORE_COURSE_A](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/CORE_COURSE_A)) , next to links to slides or other materials. Want to add that to this PR?

doctorrobinson committed:
Update README.md

updated fair links

doctorrobinson commented:
> Haven't had a chance to proof this, but I just realized that when I added course video links to the ToC in the readme I should have also added them within the readme of each course's own folder (for example https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/CORE_COURSE_A) , next to links to slides or other materials. Want to add that to this PR?

I'm not a PR pro but it looks like I need a separate one per each course, so I'm going about and doing them as well.

terichadbourne commented:
Hey @doctorrobinson This looks good to me and I'm going to merge it, then proof your other PRs. There are ways to gather up all of these edits into a single PR if you're comfortable working from your local device and then pushing to GitHub, but like you I don't think I know how to edit multiple files in a single PR if working solely through the web UI.
